README.md

```diff
@@ -12,36 +14,59 @@ Enforcing a consistent approach to start up new projects at Fusion Alliance.
 - [License](#license)
 
 ## Objectives
+
 - Maintain consistent tooling across projects
 - Enforce standards
 - Accelerate project start-up
+- Outline delivery quality processes
 
 ## Features
+
 - Code formatting enforced with EditorConfig
 - JS linting with ESLint + AirBnB
 - Sass (SCSS) + Stylelint > CSS
+- Markdown linting
+- Delivery quality [processes + templates](./docs/)
 
 ## Roadmap
-A separate [Roadmap doc](ROADMAP.md) has been created to manage the phases of development on this starter kit.
+
+A separate [Roadmap doc](ROADMAP.md) has been created to manage the phases of
+development on this starter kit.
 
 ## Requirements
-- A code editor that supports EditorConfig, or [install the EditorConfig extension for you code editor](https://editorconfig.org/#download)
+
+- A code editor that supports EditorConfig, or
+  [install the EditorConfig extension for you code editor](https://editorconfig.org/#download)
 
 ## Usage
 
+### Scripts
+
+- **Install server and website**: `npm install`
+- **Develop locally**: `npm run dev`
+- **Build**: `npm run build`
+- **Watch for lint**: `npm run dev`
+- **Run tests**: `npm test`
+
 ### SCSS
+
 - Uses the [The 7-1 Pattern](https://sass-guidelin.es/#the-7-1-pattern) for architecture
-- Generally follows [Sass Guidelines](https://sass-guidelin.es/#the-7-1-pattern) for code style
+- Generally follows [Sass Guidelines](https://sass-guidelin.es/#the-7-1-pattern)
+  for code style
 - See the [SCSS README](assets/scss/README.md) for more information
 
+### Image Minification
 
-## FAQ
+- Currently requires libpng, which you can install via homebrew `brew install libpng`
```

Comment:
I can't find a solid Windows installer, meaning people will have to build the C files themselves. https://sourceforge.net/projects/libpng/files/ is the best downloadable source. Is it possible to have a short (<10 steps) set of instructions for building libpng on Windows?

Reply:
Maybe this gnuwin build? http://gnuwin32.sourceforge.net/packages/libpng.htm

Reply:
I think that is ok? sourceforge is always a little bit shady imo but probably fine.
Is there no other package we could use?
I dont even see where `libpng` is being utilized in this project

Reply:
I dont actually see any issues with this in windows i wonder if it might be only an issue on mac
```suggestion
- Currently requires libpng on OSX, which you can install via homebrew `brew install libpng`
```